src/content/en/showcase/_index.yaml

```diff
@@ -9,33 +9,38 @@ landing_page:
 
   rows:
 
-  - heading: AliExpress increases conversion rate for new users by 104% with new Progressive Web App
+  - heading: Building the Google I/O 2016 Progressive Web App
     classname: devsite-landing-row-67 devsite-landing-row-no-image-background
     items:
     - description: >
         <p>
-          "Smarter shopping, better living!" is the motto of AliExpress, a
-          website where shoppers can buy everything from baby clothes to
-          refrigerators directly from China. Part of the Alibaba Group, the
-          global online retail marketplace is now a popular e-commerce site in
-          America, Russia, and Brazil.
+          Learn how we built a the Google I/O 2016 Progressive Web App (IOWA)
+          using web components, Polymer, and material design and launched it 
+          into production on Google.com.
         </p>
-        <blockquote>
-          "One of the reasons we built a Progressive Web App was to be able to
-          invest in the web experience across all browsers. Not only did we
-          see huge benefits on browsers that support the latest features, but
-          we also got to see a bump across the board. That is the sign of a
-          great investment, and one that will keep paying for itself as
-          browsers evolve." <br>
-          <b>Lijun Chen</b>, director of AliExpress Mobile Team.
-        </blockquote>
-      image_path: 2016/images/aliexpress/featured.png
+        <p>
+          IOWA is a single page application (SPA) that's built using web 
```

Comment:
IOWA is a single page application (SPA) that's built using web components, Polymer, Firebase, and has an extensive backend written in App Engine (Go). It pre-caches content for offline users using a service worker, dynamically loads new pages, gracefully transitions between views, and reuses content after first load.